hack/build-go.sh

```diff
@@ -10,7 +10,11 @@ STARTTIME=$(date +%s)
 OS_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${OS_ROOT}/hack/common.sh"
 source "${OS_ROOT}/hack/util.sh"
-os::log::install_errexit
+source "${OS_ROOT}/hack/lib/util/trap.sh"
```

Comment:
Would it be weird to `source "${OS_ROOT}/hack/lib/util/trap.sh"` in `hack/lib/log/stacktrace.sh` and make `os::log::stacktrace::install` invoke `os::util::trap::init_err` (which appears to be idempotent)?